src/utils/styleUtils.js

```diff
@@ -132,3 +135,11 @@ export const highlightPointStyle = new Style({
         stroke: highlightedStroke,
     }),
 })
+
+// Change a distance according to the change of DPI (from the old geoadmin)
+export function adjustDistance(dist, dpi) {
+    if (!dist) {
+        return
+    }
+    return (dist * 90) / dpi
```

Comment:
Do you know the reason of the `90` factor ?

Reply:
After some digging, the first time it's introduced is here https://github.com/geoadmin/mf-geoadmin3/pull/3280

It seems because we use different DPI for WMS and WMTS (using 96, although 90.7 is the WMTS standard) and in the print we use 150 or 254.

But for some unknown reason, it use 90 instead 96.

Should I change it to 96?

Reply:
@ismailsunni It is worse to try with 96 to see how it looks. In any case it would be good to comment this factor in code.

Reply:
@ismailsunni it might be also good to rename this function, from my understanding it is more like a width adjustment as a distance